week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -24,21 +24,22 @@ function greaterThan10() {
 
 function sortArray() {
   let letters = ["a", "n", "c", "e", "z", "f"];
-  let sortedLetters;
+  let sortedLetters = letters.sort();
 
   return sortedLetters;
+  //console.log(sortedLetters)
 }
 
 function first5() {
   let numbers = [1, 2, 3, 4, 5, 6, 7, 8];
-  let sliced;
+  let sliced = numbers.slice(0, 5);
 
   return sliced;
 }
 
 function get3rdIndex(arr) {
   let index = 3;
-  let element;
+  let element = arr[3];
```

Comment:
Almost perfect! You are repeating the number `3` and not really making use of the `index` variable though. It's really easy to make a little typo like this though 😄